Documentation/project-docs/profiling-api-status.md

```diff
@@ -16,6 +18,10 @@ The implementation of this API was making some questionable assumptions about Wi
 
 See github issue [#13153](https://github.com/dotnet/coreclr/issues/13153) for more details.
 
+### ReJIT on ARM
+
+ReJIT feature is only available on x86/x64 for now. 
```

Comment:
Lets open an issue that tracks this and link it here.

Reply:
Got it. Thanks!